packages/cloud-cognitive/src/components/Datagrid/Datagrid/DraggableElement.js

```diff
@@ -129,10 +132,10 @@ const DraggableElement = ({
           onArrowKeyDown(e, isGrabbed, index);
         }
       }}
-      onBlur={(e) => {
-        // handle when focus move to inner elements
-        setIsFocusedOnItem(e.currentTarget === e.target);
-      }}
+      // onBlur={(e) => {
+      //   // handle when focus move to inner elements
+      //   //setIsFocusedOnItem(e.currentTarget === e.target);
+      // }}
```

Comment:
If this handler is no longer used it should removed.

(the less comments and less commented out code, the easier it is to parse the module)